package-examples/function-export/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: kpt-function-export-example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-functions/label-namespace
```

Comment:
I don't remember seeing this function `label-namespace` in the catalog. What is it expected to do (assuming it existed :)) [may be this is answered later in the PR..]

Reply:
I am not sure what it does outside of seeing what it does to the config.  We can also just replace it with something else.  Could set-label with the right GVK filter.

Reply:
So this function exists in old catalog. @mengqiy might know about it. Overall, I think using the functions from function-catalog will help in users discovering these functions. `set-label` should work here. I am guessing the policy check will be related to use of invalid labels or something.